### ISSUE TITLE
Resources not found

### ISSUE THREAD

**Issue by sxamit** [channelcat/sanic-openapi#10]
this is my create_app method:
```
from sanic import Sanic
from sanic_openapi import swagger_blueprint, openapi_blueprint

def create_app():
    app = Sanic(__name__)

    app.blueprint(openapi_blueprint)
    app.blueprint(swagger_blueprint)

    from .api import v1
    app.blueprint(v1)

    return app
```

/swagger page can't load the resources. It tries to get them from: http://localhost:8000/css/typography.css
but actually they are at
http://localhost:8000/swagger/css/typography.css

Am I doing something wrong?

**Comment by sxamit** [comment]
The problem was that I tried the endpoint /swagger and not /swagger/ . Any chance to fix this?

**Comment by Typhon66** [comment]
@sxamit Should be fixed in 0.3.0
